src/components/organisms/NavBarSchedule/NavBarSchedule.tsx

```diff
@@ -167,14 +155,25 @@ export const NavBarSchedule: React.FC<NavBarScheduleProps> = ({
       eventTimeAndOrderComparator
     );
 
+    const currentVenueBookMarkEvents = eventsFilledWithPriority.filter(
+      (event) =>
+        event.isSaved && event.venueId === currentVenue?.id?.toLowerCase()
+    );
+
+    const currentVenueEvents = eventsFilledWithPriority.filter(
+      (event) => event.venueId === currentVenue?.id?.toLowerCase()
```

Comment:
`currentVenueId` constant like above

Reply:
fix in fc6549ba7fe1cff05c89fdbbd261def4decd3b54

Reply:
fix in c85b1f92ed600fc632d74bac6c9a9afb99f16203